cargo-quickinstall/tests/integration_tests.rs

```diff
@@ -36,9 +47,9 @@ fn do_dry_run_for_ripgrep() {
         target: "x86_64-unknown-linux-gnu".to_string(),
     };
 
-    let result = do_dry_run_curl(&crate_details);
+    let result = do_dry_run_curl(&crate_details).unwrap();
 
-    let expected_prefix = r#""curl" "--user-agent" "cargo-quickinstall client (alsuren@gmail.com)" "--location" "--silent" "--show-error" "--fail" "https://github.com/cargo-bins/cargo-quickinstall/releases/download/ripgrep-13.0.0-x86_64-unknown-linux-gnu/ripgrep-13.0.0-x86_64-unknown-linux-gnu.tar.gz" | "tar" "-xzvvf" "-" "-C""#;
+    let expected_prefix = "curl --user-agent cargo-quickinstall client (alsuren@gmail.com) --location --silent --show-error --fail https://github.com/cargo-bins/cargo-quickinstall/releases/download/ripgrep-13.0.0-x86_64-unknown-linux-gnu/ripgrep-13.0.0-x86_64-unknown-linux-gnu.tar.gz | tar -xzvvf - -C";
```

Comment:
as discussed above, this is a breaking change. You need to at least quote the user agent string because it contains spaces and brackets.

Reply:
Done

Reply:
Ok, so I've also added quotation for brackets by taking some code from [`shellecape`](https://docs.rs/shell-escape/latest/src/shell_escape/lib.rs.html#17-23).